### PR TITLE
PI System Deployment Samples - On Prem

### PR DESCRIPTION
**Pull request by dcbroad3**

### CODE REVIEW

File: PI-System-Deployment-Samples/OnPrem/Verify-PIServer.ps1 [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Verify test install completed successfully
+$sqlservice = Get-Service -DisplayName "SQL Server (SQLExpress)"
```
Comment (derekendres) [inline PI-System-Deployment-Samples/OnPrem/Verify-PIServer.ps1:3]: whats this file used for?  maybe need a word in the readme that you can use this one to test if the isntall was successful

File: PI-System-Deployment-Samples/OnPrem/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+[![Build Status](https://dev.azure.com/osieng/engineering/_apis/build/status/product-readiness/PI-System/Deployment_OnPrem?branchName=master)](https://dev.azure.com/osieng/engineering/_build/latest?definitionId=1640&branchName=master)
+
+This sample uses PowerShell to install Microsoft SQL Server Express, the PI Server including PI Data Archive and PI AF Server, and/or a generic self-extracting PI install kit. The script only installs the packages that are specified by flags, so it can be used to run all three installs or only one.
+
+Developed using PowerShell 5.1
+
+## Requirements
+
+- Powershell 5+
+- Install kits for products to be installed
```
Comment (derekendres) [inline PI-System-Deployment-Samples/OnPrem/README.md:14]: Which versions of the install kits work?  Or that we have tested with at least?

Comment (dcbroad3) [reply PI-System-Deployment-Samples/OnPrem/README.md:14]: Added this info just above in the README

File: PI-System-Deployment-Samples/OnPrem/Verify-PIServer.ps1 (hunk 1 above)
Comment (dcbroad3) [reply PI-System-Deployment-Samples/OnPrem/Verify-PIServer.ps1:3]: Added a "Verify" section to the README describing that script, and an extra comment in the script itself describing what it does.

**Review by derekendres — APPROVED** [review]

**Pull request merged**